man/run0.xml

```diff
@@ -167,6 +167,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--login</option></term>
+        <term><option>-i</option></term>
```

Comment:
exposing this via -i/--login, is that really the right choice? i mean, the name suggests it was about login shell semantics, but we enforce those anyway, hence the option is redundant in that sense.

Maybe just name this switch "--via-shell"?

Reply:
This is exactly the same semantics as sudo. And it additionally changes working dir to $HOME of target user. So I think `-i` is the right switch. The fact that run0 always enables login shell semantics is noted below.

Reply:
hmm, so what about this: support --login for compat, but make it an alias for --via-shell, which we advertise as the primary way in man page and --help text. with -i being advertised as short for --via-shell.

compat with sudo is nice, but I think having a reasonable, explanatory interface is even nicer. And we can have both. here ;-)

Reply:
But it isn't sufficiently explanatory since it misses the chdir part :/ I originally named the option that way but it felt off.

Reply:
hmm, what about this then: introduce new individual flags `--via-shell` + `--chdir-home` (the latter as a "special flavour" if you so will of the existign `--chdir=`. Then advertise these two independently. On top of that advertise `-i` as a shortcut for `--via-shell --chdir-home`. Never advertise `--login` but support it for compat with sudo